scripts/set-react-version.js

```diff
@@ -38,6 +38,14 @@ const profiles = {
     "react-native-macos": undefined,
     "react-native-windows": "^0.64.11",
   },
+  0.65: {
+    "hermes-engine": "~0.8.1",
+    react: "17.0.2",
+    "react-native": "^0.65.0-0",
+    "react-native-codegen": "0.0.7",
```

Comment:
For whenever 0.65 is released:
```suggestion
    "react-native": "^0.65.0",
```

Reply:
Seems like `react-native-codegen` will be necessary for 0.65. 😞